LiteCore/Query/SQLiteQuery.cc

```diff
@@ -445,20 +445,24 @@ namespace litecore {
                 case SQLITE_BLOB:
                     {
                         if ( i >= _query->_1stCustomResultColumn ) {
-                            slice        fleeceData{col.getBlob(), (size_t)col.getBytes()};
-                            Scope        fleeceScope(fleeceData, _sk);
-                            const Value* value = Value::fromTrustedData(fleeceData);
-                            if ( !value )
-                                error::_throw(error::CorruptRevisionData,
-                                              "SQLiteQueryRunner encodeColumn parsing fleece to Value failing");
-                            enc.writeValue(value);
+                            slice fleeceData{col.getBlob(), (size_t)col.getBytes()};
+                            if ( fleeceData.empty() ) {
+                                enc.writeNull();
```

Comment:
why not return false as we do for "case SQLITEE_NULL" ? 
CC: @snej

Reply:
Returning false will make it appear as MISSING, and now it appears as NULL.

Reply:
A SQLite NULL means MISSING. An empty blob (returned from a query) means a JSON/Fleece null.